### ISSUE TITLE
Generate hooks for stories

### ISSUE THREAD

**Issue by MyrionPhoenixmoon** [tschoppi/starsystem-gen#39]
This would be the first major thing away from the rules...
Ideally, we should be able to generate things like space stations, civilizations / inhabited planets etc.

We'd have to devise our own system for that, though. Which should be awesome! :D

**Comment by tschoppi** [comment]
Yeah, that sounds like a great idea! There are some rules about this for preinhabitation and stuff... Will need to look that up.

**Comment by tschoppi** [comment]
There's also GURPS City Stats, which would probably fit in nicely with all this.
